README.md

```diff
@@ -0,0 +1,18 @@
+# serilog-sinks-influxdb
+A serilog sink that writes events to InfluxDB.
```

Comment:
Maybe says that it use syslog format and referencing [influxdb docs](https://www.influxdata.com/blog/writing-logs-directly-to-influxdb/)

Reply:
added in PR #8